dali/pipeline/data/buffer.h

```diff
@@ -38,6 +38,15 @@ class GPUBackend;
 class CPUBackend;
 
 
+enum class BatchState {
+  Default = 0,
```

Comment:
...and what does this mean? We can have a `Default`, but it should be an alias for something meaningful - or, if the default behaviour is too complex, we can describe it in a comment at least.

Reply:
Added a comment here.